src/utils/duration.js

```diff
@@ -57,16 +53,13 @@ export function millisecondsToDHMS(numericDuration) {
 }
 
 export function getPreciseDuration(numericDuration) {
-    let result;
-
     const days = toDoubleDigits(Math.floor((numericDuration) / (24 * 60 * 60 * 1000)));
     let remaining = (numericDuration) % (24 * 60 * 60 * 1000);
     const hours = toDoubleDigits(Math.floor((remaining) / (60 * 60 * 1000)));
     remaining = (remaining) % (60 * 60 * 1000);
     const minutes = toDoubleDigits(Math.floor((remaining) / (60 * 1000)));
     remaining = (remaining) % (60 * 1000);
     const seconds = toDoubleDigits(Math.floor((remaining) / (1000)));
-    result = `${days}:${hours}:${minutes}:${seconds}`;
 
-    return result;
+    return `${days}:${hours}:${minutes}:${seconds}`;
 }
```

Comment:
Hi @JonanOribe, Please consider simplifying the method code to the following one

```javascript
function getPreciseDuration(value){
  const ms = value || 0;
  return [
    toDoubleDigits(Math.floor(normalizeAge(ms / ONE_DAY))),
    toDoubleDigits(Math.floor(normalizeAge((ms % ONE_DAY) / ONE_HOUR))),
    toDoubleDigits(Math.floor(normalizeAge((ms % ONE_HOUR) / ONE_MINUTE))),
    toDoubleDigits(Math.floor(normalizeAge((ms % ONE_MINUTE) / ONE_SECOND)))
  ].join(":");
};
```

Reply:
Nice refractor. The changes are done and all test are green, so LGFM